hal/src/rtl872x/deviceid_hal_impl.h

```diff
@@ -46,6 +46,8 @@
 #define WIFI_OUID_SIZE                  3
 #define DEVICE_ID_PREFIX_SIZE           6
 
+#define MODEL_VARIANT_PHOTON_2 1
```

Comment:
Is this the correct spot for this model variant definition to be defined? Are we ok starting with `1` and not `0`?